src/main/java/org/mule/extension/api/introspection/property/DisplayModelProperty.java

```diff
@@ -4,21 +4,37 @@
  * license, a copy of which has been included with this distribution in the
  * LICENSE.txt file.
  */
-package org.mule.extension.api.introspection.property.display;
 
+package org.mule.extension.api.introspection.property;
+
+import org.mule.extension.api.annotation.param.display.Password;
+import org.mule.extension.api.annotation.param.display.Text;
 import org.mule.extension.api.introspection.ParameterModel;
 
 /**
- * Represents a {@link ParameterModel} capability to be located in the extension configuration window.
+ * Represents the {@link ParameterModel} properties related to its UI.
  *
  * @since 1.0
  */
-public interface PlacementModelProperty
+public interface DisplayModelProperty
 {
+
+    String KEY = "DISPLAY.MODEL.PROPERTY";
+
+    /**
+     * The name that will be used for displaying
+     */
+    String getDisplayName();
+
     /**
-     * A unique key to identify this model property from others
+     * Returns whether the field is annotated with {@link Password} or not
```

Comment:
change this javadoc. it shouldn't refer to annotations, it should only refer to what it means, since you can always use this programatically without the intervention of the annotations